internal/domain/pull_request.go

```diff
@@ -0,0 +1,30 @@
+package domain
+
+type PullRequestEvent struct {
+	Action      string      `json:"action"`
+	PullRequest PullRequest `json:"pull_request"`
+	Repository  Repository  `json:"repository"`
+}
+
+type GitHubUser struct {
+	Login string `json:"login"`
+	Id    int    `json:"id"`
+	Url   string `json:"url"`
+}
+
+type PullRequest struct {
+	Id      int        `json:"id"`
+	Number  int        `json:"number"`
+	Title   string     `json:"title"`
+	Body    string     `json:"body"`
+	User    GitHubUser `json:"user"`
```

Comment:
You might want to define methods to handle basic operations or data retrieval for the structs (e.g., a method that returns the full URL for a pull request). This will improve maintainability and encapsulation.